server/command/command.go

```diff
@@ -49,16 +49,75 @@ func NewHandler(args *model.CommandArgs, conf *config.Config, a splunk.Splunk) H
 
 // GetSlashCommand returns command to register
 func GetSlashCommand() *model.Command {
+	splunk := model.NewAutocompleteData(
+		slashCommandName, "[alert|auth|help|log]", "connect to and interact with splunk.")
+	addSubCommands(splunk, true)
+
 	return &model.Command{
 		Trigger:          slashCommandName,
 		DisplayName:      slashCommandName,
 		Description:      pluginDescription,
 		AutoComplete:     true,
+		AutocompleteData: splunk,
 		AutoCompleteDesc: autoCompleteDescription,
 		AutoCompleteHint: autoCompleteHint,
 	}
 }
 
+func addSubCommands(splunk *model.AutocompleteData, optInstance bool) {
+	splunk.AddCommand(createAlertCommand(optInstance))
+	splunk.AddCommand(createAuthCommand(optInstance))
+	splunk.AddCommand(createHelpCommand(optInstance))
+	splunk.AddCommand(createlogCommand(optInstance))
+}
+
+func createAlertCommand(optInstance bool) *model.AutocompleteData {
+	alert := model.NewAutocompleteData(
+		"alert", "--subscribe", "subscribe to alert")
+
+	flag := []model.AutocompleteListItem{
+		{HelpText: "Subscribe", Item: "--subscribe"},
+	}
+	alert.AddStaticListArgument("subscribe to alert", true, flag)
```

Comment:
Can this label and other similar labels be made capitalized like this?

```suggestion
	alert.AddStaticListArgument("Subscribe to alert", true, flag)
```

Reply:
Done